python/res/job_queue/job_queue_node.py

```diff
@@ -112,7 +113,7 @@ def _job_monitor(self, driver, max_submit):
 
             self._set_thread_status(ThreadStatus.DONE)
 
-    def run(self, driver, max_submit=2):
+    def run(self, driver, pool_sema, max_submit=2):
```

Comment:
Could we have the same order of the arguments for this function and _job_monitor (https://github.com/equinor/libres/pull/801/files#diff-e3f501a2683b22d60135748a146d8f91R86)

Reply:
:+1: